### PR TITLE
 [fix bug 1173578] Preserve Google Analytics parameters in URL. 

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@alexgibson r? :)

File: careers/careers/static/js/filters.js [hunk 1]
```diff
@@ -85,8 +85,20 @@
 
                 // Build a querystring from populated filters.
                 var querystring = $.param(filters)
```
Comment (alexgibson) [inline careers/careers/static/js/filters.js]: Missing semi-colon here

File: careers/careers/static/js/filters.js [hunk 2]
```diff
@@ -85,8 +85,20 @@
...
-                if (!$.isEmptyObject(filters)) {
-                    querystring = "?" + querystring;
+
+                // Preserve Google Analytics parameters.
+                var ga_parameters = window.location.search.substr(1).split('&').filter(
+                  function(parameter) {
```
Comment (alexgibson) [inline careers/careers/static/js/filters.js]: Nit: use 4 space indentation for consistency

File: careers/careers/static/js/filters.js [hunk 3]
```diff
@@ -85,8 +85,20 @@
...
+                    return parameter.indexOf('utm_') === 0;
+                  }
+                );
+                if (querystring.length && ga_parameters.length) {
+                  querystring += '&';
+                }
+                querystring += ga_parameters.join('&');
```
Comment (alexgibson) [inline careers/careers/static/js/filters.js]: Could this go inside the conditional on the line above?

**Comment by alexgibson** [comment]
Tested locally and seems to be working well, just a couple of small nits but otherwise r+wc :cake:

File: careers/careers/static/js/filters.js (hunk 3 above)
Comment (glogiotatidis) [inline careers/careers/static/js/filters.js]: That wouldn't work if querystring is empty but we do have ga_parameters.

**Comment by glogiotatidis** [comment]
Thanks!

File: careers/careers/static/js/filters.js (hunk 3 above)
Comment (alexgibson) [inline careers/careers/static/js/filters.js]: Ah of course, makes sense!

**Pull request merged**